src/core_plugins/kibana/public/discover/components/field_chooser/lib/detail_views/string.html

```diff
@@ -5,12 +5,12 @@
       <span ng-if="!field.details.error">
         <a
           href=""
-          ng-show="!indexPattern.metaFields.includes(field.name)"
+          ng-show="!field.indexPattern.metaFields.includes(field.name) && !field.scripted"
```

Comment:
I also fixed a bug where metaFields were displaying the link for creating an exists filter because `indexPattern` did not exist in the local scope.